test/support/test_helpers.ex

```diff
@@ -19,17 +21,18 @@ defmodule Benchee.TestHelpers do
   end
 
   def assert_received_exactly(expected) do
-    Enum.each(expected, fn(message) -> assert_received ^message end)
+    Enum.each(expected, fn message -> assert_received ^message end)
 
     expected
-    |> Enum.uniq
-    |> Enum.each(fn(message) -> refute_received(^message) end)
+    |> Enum.uniq()
```

Comment:
:scream:

Reply:
haha. Embrace your new future, full of parentheses!